<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-I0292P0" failures="1" tests="6" name="Smoke Tests" time="17.071" errors="0" timestamp="2025-10-23T11:22:58 AKDT">
  <testcase classname="com.itg.training.lancome.FirstSeleniumTest" name="verifyLogo" time="0.147"/>
  <testcase classname="com.itg.training.lancome.FirstSeleniumTest" name="clickSignInRegister" time="4.379"/>
  <testcase classname="com.itg.training.lancome.FirstSeleniumTest" name="verifyWelcomeMessage" time="0.076"/>
  <testcase classname="com.itg.training.lancome.FirstSeleniumTest" name="loginWithValidCredentials" time="0.335">
    <failure type="java.lang.AssertionError" message="Login test failed!">
      <![CDATA[java.lang.AssertionError: Login test failed!
at com.itg.training.lancome.FirstSeleniumTest.loginWithValidCredentials(FirstSeleniumTest.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- loginWithValidCredentials -->
  <testcase classname="com.itg.training.lancome.FirstSeleniumTest" name="verifyUserLoggedIn" time="0.0">
    <skipped/>
  </testcase> <!-- verifyUserLoggedIn -->
  <testcase classname="com.itg.training.lancome.FirstSeleniumTest" name="verifyAccountWelcomeText" time="0.0">
    <skipped/>
  </testcase> <!-- verifyAccountWelcomeText -->
</testsuite> <!-- Smoke Tests -->
